### ISSUE TITLE
mc 1.12.2: Shulkerbox inside of Deathchest breaks if you break the DC to get items

### ISSUE THREAD

**Issue by MineCrak** [TheRandomLabs/VanillaDeathChest#17]
Minecraft: 1.12.2
Forge: 2838
VDC: 1.9.3.0

Hello,

A few of the people on my server have lost their shulker boxes due to this problem. 

We have used an old version, "1.12.2-1.1.1.1", for a very long time because newer ones had stopped working right with the Cubic Chunks Height/Depth mod. This shulkerbox problem had never occurred with that version. 

Recently-ish, I tested vdc 1.9.3.0 with cc and discovered that it works properly with Cubic Chunks again! Thank you for that btw. Other than it duplicating fallen heads it seemed to work fine so I updated to it on the server. 

The problem for this issue has been duplicated in single player too with VDC being the only mod, no cc or anything else. I will quote a report given to me by one of our members, TwistedZero:

```
Reproducing the glitch is as simple as breaking a death chest with a shulker box inside.
The glitch happens when breaking the death chest not when the death chest is spawned.
The glitch does not occur if the Vanilla Death Chest option 'Drop death chest' is set to 'true'. 
Picking the death chest up with CarryOn and putting it back down causes the glitch to not occur
```

Thank you for your mod. Pretty much every Cubic Chunks server I know of uses v1.1.1.1 or earlier of your mod as it has been one of the only such solutions we could always count on to work properly at all non-vanilla heights and depths. It feels good to know that we can use new versions again. 

* Also; I know there is a newer version of VDC, but there wasn't any mention of this problem being fixed in the changelog so I posted this issue in case.

**Comment by TheRandomLabs** [comment]
This was a minor oversight on my part. Fixed in 1.12.2-1.10.3.1!

**Comment by MineCrak** [comment]
Thanks!
